ghost-cat-app/src/app/View/register-page/register-page.component.ts

```diff
@@ -8,12 +9,23 @@ import {Router} from '@angular/router';
 })
 export class RegisterPageComponent implements OnInit {
 
-  constructor(private router:Router) { }
+  constructor(private router:Router, private server: ServerFacade) { }
 
   ngOnInit(): void {
   }
 
   goToPage(pageName:string):void{
     this.router.navigate([`${pageName}`]);
   }
+
+  public sendRegisterRequest(event: Event): void {
```

Comment:
Same note here as in the `html` file for this component. This code is all good and well but should be moved do a different component because it doesn't belong on our landing page